main/src/ca/uwaterloo/flix/language/phase/Reducer.scala

```diff
@@ -223,7 +223,7 @@ object Reducer {
         val taskList1 = tpe match {
           case Void | AnyType | Unit | Bool | Char | Float32 | Float64 | BigDecimal | Int8 | Int16 |
                Int32 | Int64 | BigInt | String | Regex | Region | Enum(_) | RecordEmpty |
-               Native(_) => taskList
+               Native(_) | JvmConstructor(_) | JvmMethod(_) => taskList
```

Comment:
throw internal compiler exception

Reply:
Which location do we consider?

Reply:
Nevermind, these are no longer MonoTypes so I just deleted the cases.